includes/wikia/models/DesignSystemSharedLinks.class.php

```diff
@@ -50,12 +50,12 @@ private function getLangWithFallback( $lang ) {
 
 	private $hrefs = [
 		'default' => [
-			'fan-communities' => 'http://fandom.wikia.com/explore',
+			'explore-wikis' => 'http://fandom.wikia.com/explore',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "explore-wikis" 11 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)